core/src/main/java/de/jplag/JPlag.java

```diff
@@ -71,6 +72,12 @@ public static JPlagResult run(JPlagOptions options) throws ExitException {
 
         // Compare valid submissions.
         JPlagResult result = comparisonStrategy.compareSubmissions(submissionSet);
+
+        // Use Match Merging against obfuscation
+        if (options.mergingParameters().enable()) {
```

Comment:
Tiny detail I just noticed: We could rename this to `enabled` (with a d), then it matches the clustering options.